src/Http/Requests/DeleteMediaRequest.php

```diff
@@ -24,6 +25,10 @@ public function validateResolved(): void
             throw new NotFoundHttpException(__('validation.exceptions.not_found', ['entity' => 'Media']));
         }
 
+        if (str_contains($this->media->name, 'preview')) {
+            throw new UnprocessableEntityHttpException('You can not delete a preview media');
```

Comment:
let's use bad request status code instead and please move exception message to the translations